test/blocks/article-header/article-header.test.js

```diff
@@ -0,0 +1,85 @@
+import { readFile } from '@web/test-runner-commands';
+import { expect } from '@esm-bundle/chai';
+import sinon from 'sinon';
+
+import { setConfig, getConfig } from '../../../libs/utils/utils.js';
+import { delay, waitForElement } from '../../helpers/waitfor.js';
+
+const locales = { '': { ietf: 'en-US', tk: 'hah7vzn.css' } };
+const conf = { locales, miloLibs: 'http://localhost:2000/libs' };
+setConfig(conf);
+const config = getConfig();
+
+document.body.innerHTML = await readFile({ path: './mocks/body.html' });
+const { default: init } = await import('../../../libs/blocks/article-header/article-header.js');
+
+const invalidDoc = await readFile({ path: './mocks/body-invalid.html' });
+
+describe('article header', () => {
+  const block = document.body.querySelector('.article-header');
+
+  it('creates article header block', async () => {
+    config.locale.contentRoot = '/test/blocks/article-header/mocks';
+
+    await init(block);
+    expect(document.body.querySelector('.article-category')).to.be.exist;
+    expect(document.body.querySelector('.article-title')).to.be.exist;
+    expect(document.body.querySelector('.article-author-image')).to.be.exist;
+    expect(document.body.querySelector('.article-author')).to.be.exist;
+    expect(document.body.querySelector('.article-date')).to.be.exist;
+    expect(document.body.querySelector('.article-byline-sharing')).to.be.exist;
+  });
+
+  it('should open link popup when share links are clicked', () => {
+    const shareLink = document.querySelector('.article-byline-sharing a');
+    const spy = sinon.spy(shareLink, 'click');
+    shareLink.click();
+    expect(spy.called).to.be.true;
+  });
```

Comment:
In this test, instead of spying on the function call, could you wait and query the pop up to see if it exists? That would be more closely testing the functionality surfaced to the end user.

Reply:
I updated the test. Could you take a look and LMK if it's good?